includes/classes/wp-maintenance-mode-admin.php

```diff
@@ -695,9 +696,10 @@ public function subscribe_newsletter() {
 					),
 					'body'    => wp_json_encode(
 						array(
-							'slug'  => 'wp-maintenance-mode',
-							'site'  => get_site_url(),
-							'email' => $_POST['email'],
+							'slug'     => 'wp-maintenance-mode',
+							'site'     => get_site_url(),
+							'email'    => $_POST['email'],
+							'category' => get_option( 'wpmm_page_category' ),
```

Comment:
another small thing, can we add the category under data prop, i.e [ data => category => get_option ... ]

Reply:
right, thank you!